tests/integrations/test_db_integrations_sql.py

```diff
@@ -233,7 +327,52 @@ def test_db_user(self):
         super().test_db_user()
 
 
-class Test_Mssql_db_integration(_BaseIntegrationsSqlTestClass):
+class Test_Agent_Mysql_db_integration(_BaseAgentIntegrationsSqlTestClass):
+    db_service = "mysql"
+
+    @missing_feature(library="java", reason="Java is using the correct span: db.instance")
+    @bug(library="python", reason="the value of this span should be 'world' instead of  'b'world'' ")
+    def test_db_name(self):
+        super().test_db_name()
+
+    @bug(library="python", reason="the value of this span should be 'mysqldb' instead of  'b'mysqldb'' ")
+    def test_db_user(self):
+        super().test_db_user()
+
+
+############################################################
+# Mssql: Tracer and Agent validations
+############################################################
+class Test_Tracer_Mssql_db_integration(_BaseTracerIntegrationsSqlTestClass):
+    db_service = "mssql"
+
+    @missing_feature(library="python", reason="Not implemented yet")
+    @missing_feature(library="java", reason="Not implemented yet")
+    @missing_feature(library="nodejs", reason="Not implemented yet")
+    def test_db_mssql_instance__name(self):
+        """ The Microsoft SQL Server instance name connecting to. This name is used to determine the port of a named instance. 
+            This value should be set only if it’s specified on the mssql connection string. """
+        for db_operation, request in self.requests[self.db_service].items():
+            span = self._get_sql_span_for_request(request)
+            assert span["meta"]["db.mssql.instance_name"].strip(), f"Test is failing for {db_operation}"
```

Comment:
Assertion comment could be more precise : "`db.mssql.instance_name must not be empty`"

Reply:
Done!